clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -3188,8 +3196,21 @@ private int processKafkaDataMessage(
     }
 
     if (emitMetrics.get()) {
+      /**
+       * Metrics that are only emitted for the largest active version of the store.
+       */
       hostLevelIngestionStats.recordKeySize(keyLen);
       hostLevelIngestionStats.recordValueSize(valueLen);
+      if (isLeader(partitionConsumptionState)) {
+        // metrics that are only emitted in leader replica
+        if (isManifestPayload) {
+          hostLevelIngestionStats.recordLeaderLargeValueWrites();
```

Comment:
Can you please share more context on why we need these metrics? If that's for differentiating `ManifestPayload`, `chunkedPayload`, `nonChunkedPayload`, let's give better names on these metrics.

`SmallValue` and `LargeValue` got me confused and was thinking why can't we just have a single value size metrics to differentiate

Reply:
Definitely; the motivation is to get visibility of large and small value write rate for a chunking enabled store. Context: there was a data loss bug for chunking store before; we wanted to deploy a fix with the new metrics to show how bad the situation was. We ended up scanning all topics to figure out the impact.

With the bug, only large value (>950Kb) would be lost. There is value size metric to show the avg, max, P99 size, etc. What we adding here is the write rate and count.